bash_completion/mx.zsh

```diff
@@ -0,0 +1,5 @@
+# source this file to have zsh completion for mx
+
+autoload -U bashcompinit
```

Comment:
is `bashcompinit` a script somewhere?

Reply:
Just a `zsh` function to enable compatibility with bash completions.

I'd just put it in my ~/.zshrc like this
```bash
autoload -U bashcompinit
source $MX_HOME/bash_completions/mx
```

Reply:
Yes, this looks like something better kept out of repo and under direct control of the user.

Reply:
OK, then maybe you can close this issue and put a reference to https://github.com/mukel/mx-honey in the readme file

Reply:
@mukel can you please take care of that.